src/index.ts

```diff
@@ -311,8 +351,18 @@ export const jupyterCommentingPlugin: JupyterFrontEndPlugin<ICommentPanel> = {
       }
     });
 
+    let currAwareness: Awareness | null = null;
```

Comment:
Why the null=null ?

Reply:
The type of `currAwareness` is `Awareness | null`. The initial value is null.